controllers/pages/_list_toolbar.htm

```diff
@@ -18,4 +18,17 @@
         data-stripe-load-indicator>
         Delete selected
     </button>
+    <button
+        class="btn btn-default wn-icon-clipboard"
+        disabled="disabled"
+        onclick="$(this).data('request-data', { checked: $('.control-list').listWidget('getChecked') })"
+        data-request="onCopy"
+        data-request-confirm="<?= e(trans('backend::lang.list.copy_selected_confirm')); ?>"
+        data-trigger-action="enable"
+        data-trigger=".control-list input[type=checkbox]"
+        data-trigger-condition="checked"
+        data-request-success="$(this).prop('disabled', 'disabled')"
+        data-stripe-load-indicator>
+        <?= e(trans('backend::lang.list.copy_selected')); ?>
+    </button>
```

Comment:
Can you put this on the Record controller instead? We're trying to use that more as the central "kitchen sink" demo / testing area

Reply:
Done :heavy_check_mark: